pkg/products/monitoring/prometheusRules.go

```diff
@@ -299,6 +299,23 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "pod-distribution",
```

Comment:
Could the alert name be more descriptive? Maybe something like `MultiAZPodDistribution` or `RhoamMultiAZPodDistribution` etc.

Reply:
Updated now. Thanks